main.tex

```diff
@@ -149,5 +152,10 @@ \section{Second Section}
         \texttt{\textbackslash pause}.
     \end{frame}
 
-
+\section{Bibliography}
+\nocite{einstein, knuthwebsite, latexcompanion}
+\begin{frame}{Bibliography}
+\bibliographystyle{plain}
+\bibliography{bibliography_file}
+\end{frame}
```

Comment:
Another picky comment: Could we fix the indentation here to match the rest of the file?